src/components/docs/Cards/ColorCard/styles.module.css

```diff
@@ -146,13 +160,13 @@ large-tablet-breakpoint-query and desktop-breakpoint-max-query {
   }
 }
 
-@media small-tablet-breakpoint-max-query,
+@media mobile-breakpoint-query and small-tablet-breakpoint-max-query,
 large-tablet-breakpoint-query and desktop-breakpoint-max-query {
   .root.snippetTab {
     padding-right: 10px;
     padding-left: 10px;
   }
-
+  
```

Comment:
Can you add a rule in your IDE to remove all trailing whitespace on save?

VSCode - https://stackoverflow.com/a/30884298/1470446
Sublime - https://blog.revathskumar.com/2012/08/sublimetext-remove-tailing-spaces-on-file-save.html

Reply:
Yup. Will do that tonight